phpcs.xml

```diff
@@ -2,7 +2,7 @@
 <ruleset name="FreshRSS Ruleset">
 	<description>Created with the PHP Coding Standard Generator. https://edorian.github.com/php-coding-standard-generator/</description>
 	<!-- to circumvent https://github.com/squizlabs/PHP_CodeSniffer/pull/1404 -->
-	<!--<arg name="tab-width" value="10"/>-->
+	<arg name="tab-width" value="40"/>
```

Comment:
@Frenzie Since this workaround, have you seen anything better?
I am giving a try to https://marketplace.visualstudio.com/items?itemName=ikappas.phpcs in Visual Studio Code, and it would be nice to `<arg name="tab-width" value="40"/>`

Reply:
I haven't checked.